packages/components/tests/dummy/app/templates/components/app-footer.hbs

```diff
@@ -0,0 +1,349 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "AppFooter Component"}}
+
+<Shw::Text::H1>AppFooter</Shw::Text::H1>
+
+<section data-test-percy>
+  <div class="{{if this.showHighlight 'shw-component-app-footer-layout-highlight'}}">
+    <Shw::Text::H2>Content</Shw::Text::H2>
```

Comment:
Consider using this approach for labeling the different examples, which is more consistent with other components, reduces the visual noise, and compacts a bit the section:

```
<Shw::Flex @direction="column" as |SF|>
  <SF.Item @label="[LABEL FOR THE EXAMPLE]">
    <Hds::AppFooter as |AF|>
    ...
    </Hds::AppFooter>
  </SF.Item>
  [... OTHER ITEMS]
</Shw::Flex>
```
<img width="1080" alt="screenshot_3110" src="https://github.com/hashicorp/design-system/assets/686239/941af2dc-4f8d-459a-9ea8-5debcc8f4cb5">